### DISCUSSION TITLE
Markdown and Latex Adaptation problem

### DISCUSSION THREAD

**Discussion by Z4R1TSU** [withastro/roadmap#943] [Proposal]
### Body

I have two problems about using this framework during using Markdown and Latex to publish my blog.

1. The list of Markdown (ie 1. 2. 3. ) is not shown as number like `1. ` `2. `, instead it is a dot, just like the `*` or `-` do.

2. It seems to not support Latex.

**Comment by stereobooster** [comment]
To support LaTeX (or rather Katex) you need to use remark/rehype plugins. Something like this https://astro-digital-garden.stereobooster.com/recipes/math-support-in-markdown/

**Reply by Z4R1TSU** [reply]
Latex Solution (Thank you for your reply, below is which fits for me, but duplicate rendering problem happened):
1. 
  ```shell
   npm install rehype-katex
   npm install remark-math
  ```
2. Modify `astro.config.mjs`
```json
import remarkMath from 'remark-math';
import rehypeKatex from 'rehype-katex';

    markdown: {
         remarkPlugins: [remarkMath],
         rehypePlugins: [rehypeKatex]
     }
```
